src/qt/qt_deviceconfig.cpp

```diff
@@ -365,6 +377,14 @@ DeviceConfig::ConfigureDevice(const _device_ *device, int instance, Settings *se
             }
             config++;
         }
+
+        /* The :: prefix is intentional. */
```

Comment:
Maybe explain _why_ it's there rather than just say that it's intended?

Reply:
I clarified it. I think the explanation I gave is good enough.